docs/release-notes/charmcraft-3.4.rst

```diff
@@ -165,13 +165,20 @@ Charmcraft 3.4.3
 Charmcraft 3.4.4
 ~~~~~~~~~~~~~~~~
 
-- `#2194 <https://github.com/canonical/charmcraft/issues/2194>`_ Charmcraft overwrites reactive charm's ``config.yaml``
+- `#2194 <https://github.com/canonical/charmcraft/issues/2194>`_ Charmcraft overwrites reactive charm's ``config.yaml``.
 
 Charmcraft 3.4.5
 ~~~~~~~~~~~~~~~~
 
 - Snap dependencies were updated to resolve `CVE-2025-43859
-  <https://www.cve.org/CVERecord?id=CVE-2025-43859>`_
+  <https://www.cve.org/CVERecord?id=CVE-2025-43859>`_.
+
+Charmcraft 3.4.6
+~~~~~~~~~~~~~~~~
+
+- The uv plugin breaks with uv 0.7.
```

Comment:
Sorry to retread this, but as-is it reads like a known issue. Is there no link to the ticket?

Reply:
There's no issue filed in charmcraft and no related craft-parts issue because @bepri fixed it before anyone could file one :-)

Reply:
Can we change it to "was breaking" so it doesn't sound like an active issue? We didn't account for a format for undocumented issues, so it's best if the wording makes sense at the time of reading.

Reply:
```suggestion
- The uv plugin was breaking with uv 0.7.
```